java/jd-gui/Portfile

```diff
@@ -0,0 +1,45 @@
+q# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               java 1.0
+PortGroup               github 1.0
+
+github.setup            java-decompiler jd-gui 1.6.4 v
+
+categories              java devel
+platforms               darwin
+supported_archs         x86_64
+license                 GPL-3
+maintainers             {@lhaeger gmx.net:lothar.haeger} openmaintainer
+
+description             JD-GUI, a standalone graphical utility that displays Java sources from CLASS files
+long_description        JD-GUI is a standalone graphical utility that displays Java source codes of ".class" \
+                        files. You can browse the reconstructed source code with the JD-GUI for instant access \
+                        to methods and fields.
+
+homepage                http://java-decompiler.github.io/
+
+checksums               rmd160  4224379fa8a97854ea8e94b5d71e4fe715ea2c32 \
+                        sha256  f10ff52e6abe19381d453391aa6cfebdda4b579a1e926aca7dc310b6d0bcda2b \
+                        size    365193
+                        
+java.version            1.8+
+java.fallback           openjdk11
+
+use_configure           no
```

Comment:
The universal variant is not likely to work and should probably be removed. 

`universal_variant no
`

Reply:
+1